include/mega/logging.h

```diff
@@ -124,8 +124,13 @@ class Logger
 {
 public:
     virtual ~Logger() = default;
+#ifdef ENABLE_LOG_PERFORMANCE
+    // Note: `time` and `source` are null in performance mode
+    virtual void log(const char *time, int loglevel, const char *source, const char *message, bool partial = false, bool requiresDirectOutput = false) = 0;
```

Comment:
shall we remove `time` and `source` since they're gonna be null anyways?

Reply:
Is `partial` really a good parameter name since all messages longer than 255 are partial, even the last chunk? Maybe invert and call it `final` instead?

Reply:
? it's actually the other way around: messages shorter than the buffer are !partial (i.e. the message continues): `outputBuffer` is only called with `continues = true` when copyToBuffer reaches the end of the buffer.

Reply:
Ok. I guess what confuses me is that the parameter name should be a descriptor of the current message but here it holds information about all messages? Say you have two chunks then the last chunk has `partial = false` even though it is still a partial message.

Reply:
I understand: partial is confusing. renaming to `continues`